lib/utils/display.js

```diff
@@ -3,18 +3,33 @@ const { log, output } = require('proc-log')
 const { explain } = require('./explain-eresolve.js')
 const { formatWithOptions } = require('./format')
 
+// This is the general approach to color:
+// Eventually this will be exposed somewhere we can refer to these by name.
+// Foreground colors only. Never set the background color.
+/*
+ * Black # (Don't use)
+ * Red # Danger
+ * Green # Success
+ * Yellow # Warning
+ * Blue # Accent
+ * Magenta # Done
+ * Cyan # Emphasis
+ * White # (Don't use)
+ */
```

Comment:
@wraithgar : could you elaborate on why black and white are not explicitly used?

I assume it's because we'll defer to the terminal's default coloring which should hopefully allow users to colorize as desired.

Reply:
> I assume it's because we'll defer to the terminal's default coloring which should hopefully allow users to colorize as desired.

It's exactly that.

Typically a terminal will configure a separate foreground and background color, discrete from the customizable 16 color palette. From what I've seen they will not also adjust black and white in the palette to be in accordance w/ the foreground/background colors.

If one wants to display with the foreground color, then print with "no" coloring.  There is also bold and dim if you need those.  In cases where you really do want the background color, you can inverse (I'd probably still avoid that though).